README.md

```diff
@@ -102,7 +102,47 @@ A preview of the VDK CLI commands:
 
 
 # Getting Started
-Installing VDK is a simple pip command. See the [Getting Started](https://github.com/vmware/versatile-data-kit/wiki/getting-started) guide to install VDK and create a data job.
+
+## Vdk-core and CLI
+
+```
+pip install vdk-core
+```
+
+This installs the core vdk package and the vdk command line interface. You can
+use them to run jobs in your local shell environment.
+
+## Running with docker, helm, and k8s
```

Comment:
The title should explain what is the value to the user or the user-facing feature/inteface , and not the implementation steps. 

So maybe. Deploy and manage jobs in a cloud like environment. 

Or just "Start VDK Control Service"